internal/datacoord/server.go

```diff
@@ -433,22 +433,23 @@ func (s *Server) initServiceDiscovery() error {
 	// TODO implement rewatch logic
 	s.dnEventCh = s.session.WatchServices(typeutil.DataNodeRole, rev+1, nil)
 
-	icSessions, icRevision, err := s.session.GetSessions(typeutil.IndexCoordRole)
-	if err != nil {
-		log.Error("DataCoord get IndexCoord session failed", zap.Error(err))
-		return err
-	}
-	serverIDs := make([]UniqueID, 0, len(icSessions))
-	for _, session := range icSessions {
-		serverIDs = append(serverIDs, session.ServerID)
-	}
-	s.icEventCh = s.session.WatchServices(typeutil.IndexCoordRole, icRevision+1, nil)
+	//icSessions, icRevision, err := s.session.GetSessions(typeutil.IndexCoordRole)
```

Comment:
you can delete this. （or in next pr）

Reply:
Keep this part of the code until we're sure we won't be using it again.